man/syncthing-config.5

```diff
@@ -555,7 +555,15 @@ delete files from other devices.  The mechanism is described in a
 .TP
 .B scanProgressIntervalS
 The interval in seconds with which scan progress information is sent to the GUI. Setting to \fB0\fP
-will cause Syncthing to use the default value of two.
+will cause Syncthing to use the default value of two. Setting a negative value will disable
```

Comment:
These docs are generated. You'll need to make a separate PR to update
https://github.com/syncthing/docs/blob/47c07a73f278bde5e3e514b7ab2c5d74d53aba56/users/config.rst?plain=1#L493